### PR TITLE
[WIP] Terraform to create ELBs for Cloudfoundry.  

### PR DESCRIPTION
**Pull request by cnelson**
@dlapiduz / @LinuxBozo is this what you were expecting? 

At minimum this probably needs security groups added to it before merge, but I wasn't sure what those should be.  The ELB configuration was copied from https://github.com/18F/cg-manifests/blob/0a597f2617d67096b66ded9e6ee007a65f2fb6d3/provisioning/elb.json but I have no idea how far out of sync that is with what you are actually running in us-east.

Feedback appreciated.

This is for 18F/cg-atlas#64

### CODE REVIEW

**Comment by dlapiduz** [comment]
I havent been doing terraform in a while so I will leave it to @LinuxBozo

**Comment by LinuxBozo** [comment]
@cnelson The security group list will only have web traffic one right now, so:
```
elb_security_groups = "${module.stack.web_traffic_security_group}"
```
Also, missing a listener for port 4443 -> 80  TCP-SSL

File: terraform/stacks/production/stack.tf [hunk 1]
```diff
@@ -14,4 +14,15 @@ module "stack" {
...
+}
+
+module "cf" {
+    source = "../../modules/cloudfoundry"
+
+    account_id = "${var.account_id}"
+    stack_description = "${var.stack_description}"
+    elb_main_cert_name = "${var.main_cert_name}"
+    elb_apps_cert_name = "${var.apps_cert_name}"
+    elb_subnets = "${module.stack.public_subnet_az1},${module.stack.public_subnet_az1}"
```
Comment (LinuxBozo) [inline terraform/stacks/production/stack.tf]: These are both az1, second should be az2

File: terraform/stacks/staging/stack.tf [hunk 2]
```diff
@@ -14,4 +14,15 @@ module "stack" {
...
+}
+
+module "cf" {
+    source = "../../modules/cloudfoundry"
+
+    account_id = "${var.account_id}"
+    stack_description = "${var.stack_description}"
+    elb_main_cert_name = "${var.main_cert_name}"
+    elb_apps_cert_name = "${var.apps_cert_name}"
+    elb_subnets = "${module.stack.public_subnet_az1},${module.stack.public_subnet_az1}"
```
Comment (LinuxBozo) [inline terraform/stacks/staging/stack.tf]: Same issue as above, az2 missing

**Comment by cnelson** [comment]
I created a story 18F/cg-atlas#66 to add the custom SSL negotiation policy as soon as terraform supports doing so.

**Comment by cnelson** [comment]
Update the stage and prod stacks to use the *.fr certs for the app ELB as well for now per @dlapiduz

**Pull request merged**